src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -149,75 +153,6 @@
                DisplayName="Embedded in DLL/EXE, portable across platforms" />
   </EnumProperty>
 
-  <StringProperty Name="LangVersion"
```

Comment:
This big diff is just the moving of these properties to later in the file and setting the category to 'Advanced'.